README.md

```diff
@@ -20,6 +20,8 @@ In this tutorial, you will:
 
 ✔ Use the KILT SDK to write onto and read from the KILT blockchain.
 
+❌ Not get in touch with our KILT anonymous credentials. But don't worry: There will be a tutorial for this very soon!
```

Comment:
AFAIK 'getting in touch' is just 'contact' in the sense of contacting someone. @Dudleyneedham ? Can you help?
I could imagine 'We will not introduce our KILT anonymous credentials just yet. But don't worry: There will be a tutorial for this very soon!'

Reply:
You are right.